src/Perl6/Pod.nqp

```diff
@@ -116,32 +124,175 @@ class Perl6::Pod {
         ).compile_time_value
     }
 
+    sub strip-quotes($val) {
+        say("DEBUG: incoming val for stripping: |$val|") if $debugp;
+        my $m := $val ~~ /^(.)(.)(.*)(.)$/;
+        my $first  := ~$m[0];
+        my $second := ~$m[1];
+        my $last   := ~$m[3];
+        if %h{$first} && $last eq %h{$first} {
+            say("DEBUG: found quote pair for val |$val|") if $debugp;
+            $val := subst($val, /^ $first /, q{});
+            $val := subst($val, / $last $/, q{});
+        }
+        elsif 'q' eq nqp::lc($first) && %h2{$second} && $last eq %h2{$second} {
+            say("DEBUG: found Q quote pair for val |$val|") if $debugp;
+            my $first2 := nqp::concat($first, $second);
+            $val := subst($val, /^ $first2 /, q{});
+            $val := subst($val, / $last $/, q{});
+        }
+
+        if $debugp {
+            say("  leading character is   [{$m[0]}]");
```

Comment:
Maybe that's too verbose? Squish it into one line perhaps?

Reply:
ok...